examples/c/CMakeLists.txt

```diff
@@ -89,13 +89,6 @@ define_c_test(
     ARGUMENTS -h $<SHELL_PATH:$<TARGET_FILE_DIR:ex_file_system>/WT_HOME>
 )
 
-define_c_test(
-    TARGET ex_key_provider
-    SOURCES ex_key_provider.c
-    DIR_NAME ex_key_provider
-    ARGUMENTS -h $<SHELL_PATH:$<TARGET_FILE_DIR:ex_key_provider>/WT_HOME>
-)
-
```

Comment:
Is it intentional to remove the example?  If this is temporary, there's probably ways to disable it running without erasing git history.

Reply:
Yes, we do want to remove this example. Disagg examples don't make much sense for the wider WT audience. Also, disagg examples would require elaborate setup before they can run, which may defeat the purpose of the example being short, self-contained, and demonstrating the usage.

It is especially true for key_provider example, which only relevant to MongoDB internals.

Reply:
Okay, thanks.